File: tatooine/src/test/java/org/example/product/tatooine/test/TatooineModuleTest.java
package org.example.product.tatooine.test;

import static org.junit.jupiter.api.Assertions.assertSame;

import org.junit.jupiter.api.Test;

public class TatooineModuleTest {

    @Test
    public void testModule() {
        assertSame("Junit4", "Junit" + (2 + 2));
    }
}

File: tatooine/src/test/kotlin/org/example/product/tatooine/test/TatooineModuleTest.kt
package org.example.product.tatooine.test

import org.junit.jupiter.api.Assertions.assertSame
import org.junit.jupiter.api.Test

class TatooineModuleTest {
    @Test
    fun testModule() {
        assertSame("Junit4", "Junit" + (2 + 2))
    }
}
